<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Stats</class>
 <widget class="QMainWindow" name="Stats">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1558</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1561</width>
      <height>681</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#234E70;

color:#FBF8BE;
</string>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QPushButton" name="return_to_main_button">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>600</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #FBF8BE;

   color:#1E2761;
}</string>
      </property>
      <property name="text">
       <string>Return to Main Menu</string>
      </property>
     </widget>
     <widget class="QPushButton" name="save_pic_button1">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>600</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #FBF8BE;

   color:#1E2761;
}</string>
      </property>
      <property name="text">
       <string>Save Pic</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="titel_Tab1">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>10</y>
        <width>901</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>22</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Statistics for all patients&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QPushButton" name="return_to_main_button_2">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>600</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #FBF8BE;

   color:#1E2761;
}</string>
      </property>
      <property name="text">
       <string>Return to Main Menu</string>
      </property>
     </widget>
     <widget class="QPushButton" name="save_pic_button_2">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>600</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #FBF8BE;

   color:#1E2761;
}</string>
      </property>
      <property name="text">
       <string>Save Pic</string>
      </property>
     </widget>
     <widget class="QLabel" name="titel_tab2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>10</y>
        <width>901</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>22</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Statistics for all patients&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="range_of_years_comboBox">
      <property name="geometry">
       <rect>
        <x>1400</x>
        <y>180</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>1382</x>
        <y>140</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;range of years&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="update_tab2">
      <property name="geometry">
       <rect>
        <x>1410</x>
        <y>220</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #FBF8BE;

   color:#1E2761;
}</string>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>tab_3</string>
     </attribute>
     <widget class="QPushButton" name="return_to_main_button_3">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>600</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #FBF8BE;

   color:#1E2761;
}</string>
      </property>
      <property name="text">
       <string>Return to Main Menu</string>
      </property>
     </widget>
     <widget class="QPushButton" name="save_pic_button_3">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>600</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #FBF8BE;

   color:#1E2761;
}</string>
      </property>
      <property name="text">
       <string>Save Pic</string>
      </property>
     </widget>
     <widget class="QLabel" name="titel_tab2_2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>10</y>
        <width>901</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>22</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Statistics for all patients&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="Obesity_grade_label">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>80</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Obesity grade</string>
      </property>
     </widget>
     <widget class="QLabel" name="Age_groups_label">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>80</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Age groups</string>
      </property>
     </widget>
     <widget class="QLabel" name="Procedure_label">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>80</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Procedure</string>
      </property>
     </widget>
     <widget class="QLabel" name="Months_after_label">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>90</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Months after the surgery</string>
      </property>
     </widget>
     <widget class="QComboBox" name="Obesity_grade_comboBox">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>110</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="Age_groups_comboBox">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>110</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="Procedure_comboBox">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>110</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="Months_after_comboBox">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>110</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="update_tab3">
      <property name="geometry">
       <rect>
        <x>1000</x>
        <y>110</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Tab 4</string>
     </attribute>
     <widget class="QLabel" name="Age_groups_label_2">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>80</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Age groups</string>
      </property>
     </widget>
     <widget class="QPushButton" name="save_pic_button_4">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>600</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #FBF8BE;

   color:#1E2761;
}</string>
      </property>
      <property name="text">
       <string>Save Pic</string>
      </property>
     </widget>
     <widget class="QComboBox" name="Obesity_grade_comboBox_tab4">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>110</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="titel_tab2_3">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>10</y>
        <width>901</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>22</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Statistics for all patients&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="Procedure_label_2">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>80</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Procedure</string>
      </property>
     </widget>
     <widget class="QPushButton" name="update_tab4">
      <property name="geometry">
       <rect>
        <x>1070</x>
        <y>110</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
     <widget class="QPushButton" name="return_to_main_button_4">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>600</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #FBF8BE;

   color:#1E2761;
}</string>
      </property>
      <property name="text">
       <string>Return to Main Menu</string>
      </property>
     </widget>
     <widget class="QComboBox" name="Age_groups_comboBox_tab4">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>110</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="Obesity_grade_label_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>80</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Obesity grade</string>
      </property>
     </widget>
     <widget class="QComboBox" name="Procedure_comboBox_tab4">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>110</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
